package.json

```diff
@@ -12,7 +13,8 @@
     "react-dom": "^16.12.0",
     "react-flip-move": "^3.0.1",
     "react-highlight.js": "1.0.7",
-    "shadow-cljs": "^2.11.7"
+    "shadow-cljs": "^2.11.18",
+    "xregexp": "^5.0.1"
```

Comment:
I'm not seeing includes for this dependency--what is it used for?

Reply:
`cuerdas` requires the npm package `xregexp` and for some reason it isn't being pulled in automatically by shadow-cljs